includes/class-wc-payments-account.php

```diff
@@ -1802,4 +1806,22 @@ private function tracks_event( string $name, array $properties = [] ) {
 
 		Logger::info( 'Tracks event: ' . $name . ' with data: ' . wp_json_encode( WC_Payments_Utils::redact_array( $properties, [ 'woo_country_code' ] ) ) );
 	}
+
+	/**
+	 * Get user data to send to the onboarding flow.
+	 *
+	 * @return array The user data.
+	 */
+	private function get_onboarding_user_data(): array {
+		return [
+			'user_id'    => get_current_user_id(),
```

Comment:
We have no guarantee of matchup. We would have to keep them in sync. But you are right, we can send the session ID fully constructed because we have the WPCOM blog ID by that point. Thanks for raising this point.